exes/mqtt-broadcaster/inc/config.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <tfc/confman.hpp>
+#include <tfc/confman/observable.hpp>
+
+// File under /etc/tfc/mqtt-broadcaster/def/mqtt_broadcaster.json which lists the signals that are supposed to be published
+// on the mqtt broker. Updating the signals in this file will cause the running program to stop and restart
+struct config {
+  std::string _mqtt_broker_address{};
+  std::string _mqtt_broker_port{};
+  std::string _mqtt_broker_username{};
+  std::string _mqtt_broker_password{};
+  std::string _sparkplug_b_edge_node_id{};
+  std::string _sparkplug_b_group_id{};
+  std::string _client_id{};
+
+  struct glaze {
+    static constexpr auto value{ glz::object("_mqtt_broker_address",
+                                             &config::_mqtt_broker_address,
+
+                                             "_mqtt_broker_port",
+                                             &config::_mqtt_broker_port,
+
+                                             "_mqtt_broker_username",
+                                             &config::_mqtt_broker_username,
+
+                                             "_mqtt_broker_password",
+                                             &config::_mqtt_broker_password,
+
+                                             "_sparkplug_b_edge_node_id",
+                                             &config::_sparkplug_b_edge_node_id,
+
+                                             "_sparkplug_b_group_id",
+                                             &config::_sparkplug_b_group_id,
+
+                                             "_client_id",
+                                             &config::_client_id
```

Comment:
Why the underscore prefix?

Reply:
My thinking was that it might be a good idea to have a specific naming convention for variables that are kept in a configuration file so that they are easily identifiable.

Reply:
changing this to be cleaner.